### PR TITLE
DOC: Update readme and delete contributing guidelines in favour of nipreps website

### PR DESCRIPTION
**Pull request by josephmje**
I also fixed a typo in the figure1 svg file but wasn't sure how to render it into png.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,6 +23,9 @@ dMRIPrep
 [`Documentation <https://www.nipreps.org/dmriprep/>`__]
 [`Support at neurostars.org <https://neurostars.org/tags/dmriprep>`__]
 
+About
+-----
+
```
Comment (oesteban) [inline README.rst:28]:
```suggestion
```

File: README.rst [hunk 2]
```diff
@@ -35,11 +41,28 @@ workflow, which can help ensure the validity of inference and the interpretabili
...
 preprocessing, and will be updated as newer and better neuroimaging software
 becomes available.
 
 dMRIPrep performs basic preprocessing steps such as head-motion correction,
 susceptibility-derived distortion correction, eddy current correction, etc.
 providing outputs that can be easily submitted to a variety of diffusion models.
+
+Getting involved
+----------------
+
```
Comment (oesteban) [inline README.rst:55]:
```suggestion
```

File: README.rst [hunk 3]
```diff
@@ -35,11 +41,28 @@ workflow, which can help ensure the validity of inference and the interpretabili
...
+We welcome all contributions!
+We'd like to ask you to familiarize yourself with our `contributing guidelines <https:/www.nipreps.org/community/CONTRIBUTING>`__.
+For ideas for contributing to `dMRIPrep`, please see the current list of `issues <https://github.com/nipreps/dmriprep/issues>`__.
```
Comment (oesteban) [inline README.rst:58]:
```suggestion
For ideas for contributing to *dMRIPrep*, please see the current list of `issues <https://github.com/nipreps/dmriprep/issues>`__.
```

File: README.rst [hunk 4]
```diff
@@ -35,11 +41,28 @@ workflow, which can help ensure the validity of inference and the interpretabili
 of results.
 
 The workflow is based on `Nipype <https://nipype.readthedocs.io>`__ and
-encompases a large set of tools from other neuroimaging package. This pipeline
+encompasses a large set of tools from other neuroimaging package. This pipeline
 was designed to provide the best software implementation for each state of
```
Comment (oesteban) [inline README.rst:45]:
```suggestion
encompasses a large set of tools from other neuroimaging packages.
This pipeline was designed to provide the best software implementation for each state of
```

**Review by oesteban — APPROVED** [review]
LGTM - a few nitpicks

**Comment by oesteban** [comment]
We should probably rebase this into master - there are many commits there that I don't think are necessary.

File: dmriprep/utils/images.py [hunk 5]
```diff
@@ -42,7 +42,7 @@ def rescale_b0(in_file, mask_file, out_path=None):
     return out_path, signal_drift.tolist()
 
 
-def median(in_file, out_path=None):
+def median(in_file, dtype=None, out_path=None):
```
Comment (josephmje) [inline dmriprep/utils/images.py:45]:
```suggestion
def median(in_file, out_path=None):
```

**Pull request merged**